SDPi_Supplement/asciidoc/volume1/use-cases/tf1-ch-c-use-case-sicdmp.adoc

```diff
@@ -0,0 +1,50 @@
+// = Use Case Feature: Standalone ICU Dashboard MP (SICDmp)
+
+[sdpi_offset=4]
+=== Use Case Feature {var_use_case_id}: Standalone ICU Dashboard Multiple Patient (SICDmp)
+
+==== Narrative:
+
+Dr. Presky is in the ICU evaluating a trauma patient from a remote location in the ICU.  The dashboard allows access to multiple patients, so he first needs to select the patient of interest.  This can be done by location or by patient name (if available).
+
+He can view previous radiology results, electrosurgical equipment settings, patient readings such as HR, Blood Pressure, SpO2 and associated waveforms integrated on his real-time ‘Dashboard’ display.  The dashboard display can display visual alarms but does not sound alerts or provide any remote-control capabilities.  (This display can be considered an xDISmp as described in Annex A.)
```

Comment:
"xDISmp as described in Annex A." see https://github.com/IHE/sdpi-fhir/pull/184/files#r1016555486

Reply:
Updated as for "sp" above.